Android/app/src/main/java/org/opendroneid/android/bluetooth/OpenDroneIdDataManager.java

```diff
@@ -58,6 +58,12 @@ void receiveDataNaN(byte[] data, int peerHash, long timeNano, LogMessageEntry lo
         receiveData(timeNano, "NaN ID: " + peerHash, peerHash, 0, message, logMessageEntry);
     }
 
+    void receiveDataWiFi(byte[] data, String mac, long macLong, long timeNano, LogMessageEntry logMessageEntry) {
+        OpenDroneIdParser.Message<?> message = OpenDroneIdParser.parseAdvertisingData(data, 1, timeNano, logMessageEntry);
+        if (message == null)
+            return;
+        receiveData(timeNano, mac, macLong, 0, message, logMessageEntry);
```

Comment:
Please add the RSSI level(instead of using 0). See the comment further below related to this.